db/migrate/20160726034132_add_last_logged_in_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastLoggedInToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :last_logged_in, :datetime
```

Comment:
Why do we need to know when a user last logged in?  Are you using this somewhere else?